index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html class="no-js" lang="en"> 
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Apple</title>
+        <script src="https://kit.fontawesome.com/1f67cb811d.js"></script>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./Assets/stylesheet.css">
+    </head>
+    <body>
+        
+            
+        
+        <main>
+                <nav>
+                    <ul>
+                        <li><a href="#"><i class="fab fa-apple"></i></a></li>
+                        <li><a href="#">Store</a></li>
+                        <li><a href="#">Mac</a></li>
+                        <li><a href="#">iPod</a></li>
+                        <li><a href="#">iPhone</a></li>
+                        <li><a href="#">iPad</a></li>
+                        <li><a href="#">iTunes</a></li>
+                        <li><a href="#">Support</a></li>
+                        <li class="search"><form action="Search">
+                            <i class="fas fa-search"></i><input type="search" aria-label="search">
```

Comment:
`SUGGESTION`

The `fas fa-search` is not rendering, it may be that you didn't link the `font awesome` assets properly, :thinking: ?